bin/audit-svg-colors.js

```diff
@@ -257,16 +279,12 @@ function findClosestColor( value, targetValues ) {
 }
 
 function findPaletteColorName( value ) {
-	let name;
-
 	// Iterating from right to make sure color name aliases aren’t caught
-	_.forInRight( PALETTE.colors, ( paletteValue, paletteColorName ) => {
-		if ( paletteValue === value ) {
-			name = paletteColorName;
-			return false;
-		}
-	} );
-
+	const name = Object.keys( PALETTE.colors ?? {} )
+		.reverse()
+		.find( ( paletteColorName ) => {
```

Comment:
I think we can still use `findLast` instead of having to call both `reverse` and `find`?